python/xorbits/_mars/constants.py

```diff
@@ -14,8 +14,10 @@
 
 import os
 
-MARS_TEMP_DIR = "/tmp/xorbits"
-MARS_TEMP_DIR_WIN = "C:\\Temp\\xorbits"
+from .utils import get_current_user
```

Comment:
Xoscar 0.0.4 released, you can try to use the identical function from it.

Reply:
Fix